src/app/security/maturity-model/components/MobileTableOfContents.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { FormListbox } from '@/components/Form/FormListbox/FormListbox'
+
+import { coreFunctionsData } from '../data/coreFunctionsData'
+import { scrollToSection } from '../utils/scrollToSection'
+import { useUrlHash } from '../utils/useUrlHash'
+
+const options = coreFunctionsData.map(({ slug, title }) => ({
+  id: slug,
+  name: title,
+}))
+
+export function MobileTableOfContents() {
+  const { updateHash, isSectionActive, getHashFromSlug } = useUrlHash()
+
+  function handleChange(slug: string) {
+    updateHash(slug)
+
+    const sectionHash = getHashFromSlug(slug)
+    scrollToSection(sectionHash)
+  }
+
+  const selectedItem = coreFunctionsData.find((item) =>
+    isSectionActive(item.slug),
+  )
+
+  return (
+    <div className="flex justify-center">
```

Comment:
This is making it center aligned when it hits `sm`?
 
![CleanShot 2024-10-22 at 14 16 14@2x](https://github.com/user-attachments/assets/e0db1399-afa6-4848-a84a-94b409befb66)